test/logging.jl

```diff
@@ -231,33 +231,30 @@ end
     @test shouldlog(logger, Info, Base, :group, :asdf) === true
     handle_message(logger, Info, "msg", Base, :group, :asdf, "somefile", 1, maxlog=2)
     @test shouldlog(logger, Info, Base, :group, :asdf) === false
+    @test catch_exceptions(logger) === false
 
     # Log formatting
     function genmsg(level, message, _module, filepath, line; kws...)
         io = IOBuffer()
         logger = SimpleLogger(io, Debug)
         handle_message(logger, level, message, _module, :group, :id,
                        filepath, line; kws...)
-        s = String(take!(io))
-        # Remove the small amount of color, as `Base.print_with_color` can't be
-        # simply controlled.
-        s = replace(s, r"^\e\[1m\e\[..m(.*)\e\[39m\e\[22m"m => s"\1")
-        # println(s)
-        s
+        String(take!(io))
     end
 
     # Simple
     @test genmsg(Info, "msg", Main, "some/path.jl", 101) ==
     """
-    [ Info: msg @ Main path.jl:101
+    ┌ Info: msg
+    └ @ Main some/path.jl:101
     """
 
     # Multiline message
     @test genmsg(Warn, "line1\nline2", Main, "some/path.jl", 101) ==
     """
-    ┌ Warning: line1
+    ┌ Warn: line1
```

Comment:
It'd be better if this also used the full word, for consistency if nothing else.

Reply:
What I did here was remove the special case when formatting the log level as part of the effort to do the least amount of formatting in `SimpleLogger` while keeping it fairly readable.

I'm leaning toward just renaming `CoreLogging.Warn` to `CoreLogging.Warning` because it seems really kind of bad to have to special case the printing of `LogLevel` every time you want to print it! How about I add the special case back in here, and make a separate PR for the renaming.

Reply:
By the way, removing the special casing was more about simplicity and making custom log levels print correctly rather than changing `Warn`. The printing in `ConsoleLogger` is unaffected, which is what users will see by default.